From 7a32af1d8f904990d016d96d50894c49686db14c Mon Sep 17 00:00:00 2001
From: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
Date: Thu, 29 May 2025 14:44:17 +0100
Subject: [PATCH 37/37] corstone1000: purge remoteproc DTS node

Purge remoteproc DTS node
This is done to remove the remote proc node from the DTS passed
to Linux from U-Boot because the device tree binding for remoteproc
has not been upstreamed yet. Existence of remoteproc DTS node in Linux
is causing dt-schema test for SystemReady-IR v2.0 certification to fail.

Signed-off-by: Harsimran Singh Tungal <harsimransingh.tungal@arm.com>
Upstream-Status: Denied [RFC: https://lore.kernel.org/u-boot/aca7e6fa-2dec-a7c5-e47e-84c5ffa6f9b7@gmx.de/T/#m16d14ee960427cc88066bdcdd76f0a26738bb66d]
---
 board/armltd/corstone1000/corstone1000.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/board/armltd/corstone1000/corstone1000.c b/board/armltd/corstone1000/corstone1000.c
index c8fcbf976e0..1b9c3a9ed16 100644
--- a/board/armltd/corstone1000/corstone1000.c
+++ b/board/armltd/corstone1000/corstone1000.c
@@ -33,6 +33,7 @@ DT_NON_COMPLIANT_PURGE_LIST(foo) = {
 	{ .node_path = "/soc/mhu@1b010000" },
 	{ .node_path = "/soc/mhu@1b020000" },
 	{ .node_path = "/soc/mhu@1b030000" },
+	{ .node_path = "/soc/remoteproc@1a010310" },
 };
 
 #define CORSTONE1000_KERNEL_PARTS 2
-- 
2.25.1

